src/components/modules/chat/hooks/useFileAttachment.hook.ts

```
@@ -0,0 +1,249 @@
+
+  // Process and add files
+  const processFiles = useCallback(async (files: FileList | File[]) => {
+    const fileArray = Array.from(files);
+    const newFiles: AttachedFile[] = [];
+
+    for (const file of fileArray) {
+      const error = validateFile(file);
+      const preview = await generatePreview(file);
+      
+      const attachedFile: AttachedFile = {
+        id: `${Date.now()}-${Math.random().toString(36).substr(2, 9)}`,
```

ESLint

[error] Insert `··`

(prettier/prettier)